addons/website_sale_stock_wishlist/models/product_wishlist.py

```diff
@@ -1,53 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import fields, models, _
-
-
-class ProductWishlist(models.Model):
-    _inherit = "product.wishlist"
-
-    stock_notification = fields.Boolean(default=False, required=True)
```

Comment:
Actually, we could make `stock_notification` be a computed field that will check `has_stock_notification` it could even have an `inverse` that would add the partner in the product's notification partner list.
It's probably a good idea in order to not break possible customisations.